cmake/config/CMakeLists.txt

```diff
@@ -205,7 +204,7 @@ SET(_files
   )
 SET(_makefiles
   ${CMAKE_BINARY_DIR}/${DEAL_II_SHARE_RELDIR}/Make.global_options
-  ${CMAKE_CURRENT_BINARY_DIR}/${DEAL_II_SHARE_RELDIR}/Make.global_options
+  ${CMAKE_CURRENT_BINARY_DIR}/Make.global_options
```

Comment:
No, this line was wrong.

I got sidetracked a bit and forgot to put this change in a separate commit.